### DISCUSSION TITLE
How to join into specific server

### DISCUSSION THREAD

**Discussion by lilobject** [ORelio/Minecraft-Console-Client#1833] [Q&A]
I know this is probably a dumb question, but I can connect to the server easily, but I need it to type /join <world name> so I go straight to that world. Is there a way I can script it so it sends that every so often? thanks!

**Comment by lilobject** [comment]
Also when it relogs, it sends me back to the main hub and not to the world directly. I have the antiafk /command as /join <world name> and it doesn't seem to be working.

**Reply by ORelio** [reply]
Make sure you set `enabled=true` to enable the functionality.

**Comment by ReinforceZwei** [comment]
To send a command to server, set the AntiAFK command as `command=/your-command`.

**Comment by ORelio** [comment]
For that, you can also use the [ScriptScheduler](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#using-the-script-scheduler--task-scheduler). Enable it in config, then create an ini file like `tasks.ini` with content like this:

```ini
[Task]
triggerOnLogin=true
action=send /login
```
This will send `/login` every time you join a server.

**Reply by Shywizz** [reply]
Where do you place scripts you make, is there a folder to make or just throw them into the same directory?

**Reply by ORelio** [reply]
As per [README](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#how-to-write-a-script-file):
> A script file can be launched by using `/script <filename>` in the client's command prompt.
The client will automatically look for your script in the current directory or `scripts` subfolder.
If the file extension is `.txt` or `.cs`, you may omit it and the client will autodectect the extension.

**Comment by Daenges** [comment]
I also made a [video tutorial](https://youtu.be/JbDpwwETEnU) about scripts. :)
